screens/Contacts/ContactListScreen.tsx

```diff
@@ -0,0 +1,125 @@
+import { useState } from 'react';
+import { View, Text, FlatList, Image, TextInput, TouchableOpacity} from 'react-native';
+import { styles } from './styles';
+import { Ionicons } from '@expo/vector-icons'
```

Comment:
Be sure to stipulate that in your PR next time. To be aware of the new dependencies.